### DISCUSSION TITLE
Some questions about deepmd parameters

### DISCUSSION THREAD

**Discussion by SEU-NiuWenLong** [deepmodeling/dpgen#1248] [Q&A] [answered]
[https://tutorials.deepmodeling.com/en/latest/CaseStudies/Mg-Y_alloy/Mg-Y_alloy.html#introduction](url), in this tutorial, the list format is used for both the pressure and temperature ranges, and sys_idx also uses the list format. Does this mean that every structure in sys_idx will be simulated by molecular dynamics at all temperatures and pressures in the list, for example, all the pressure values in the pressure list are used at 50K, and all the pressure values in the pressure list are also used at 100K?
In addition, is the number of training steps mentioned in the article about deepmd a parameter of number_steps?

**Comment by njzjz** [comment, marked as answer]
Yes.

**Reply by SEU-NiuWenLong** [reply]
OK，thanks
